src/components/Loading/README.md

```diff
@@ -54,6 +54,10 @@ class EmbeddedAppLoadingExample extends React.Component {
 }
 ```
 
+#### Deprecation rationale
```

Comment:
Should this be an h3?

Reply:
The parser tries fails because it tries to render them as code examples